### ISSUE TITLE
missing /razer/chromasdk

### ISSUE THREAD

**Issue by sharpblade4** [chroma-sdk/chroma-python#14]
Hi, 
I'm using linux (debian 8.9), and I just connected the razer black widow chroma v2 keyboard to the computer without installing any drivers (as razer doesn't supply linux drivers and I didn't want to install the community drivers). According to razer's website the SDK is installed automatically, but I guess they ment for windows only. 

Anyway, when I tried this package (code below) I got the following error:

```
import ChromaPython
Info = ChromaPython.ChromaAppInfo()
Info.DeveloperName = '1'
Info.DeveloperContact = 'a@b.com'
Info.Category = 'application'
Info.SupportedDevices = ['keyboard']
Info.Description = '1'
Info.Title = '1'
app = ChromaPython.ChromaApp(Info)
```
```
Unexpected Error!
---------------------------------------------------------------------------
ConnectionRefusedError                    Traceback (most recent call last)
~/work/python3env/lib/python3.7/site-packages/urllib3/connection.py in _new_conn(self)
    156             conn = connection.create_connection(
--> 157                 (self._dns_host, self.port), self.timeout, **extra_kw
    158             )

~/work/python3env/lib/python3.7/site-packages/urllib3/util/connection.py in create_connection(address, timeout, source_address, socket_options)
     83     if err is not None:
---> 84         raise err
     85

~/work/python3env/lib/python3.7/site-packages/urllib3/util/connection.py in create_connection(address, timeout, source_address, socket_options)
     73                 sock.bind(source_address)
---> 74             sock.connect(sa)
     75             return sock

ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

NewConnectionError                        Traceback (most recent call last)
~/work/python3env/lib/python3.7/site-packages/urllib3/connectionpool.py in urlopen(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)
    671                 headers=headers,
--> 672                 chunked=chunked,
    673             )

~/work/python3env/lib/python3.7/site-packages/urllib3/connectionpool.py in _make_request(self, conn, method, url, timeout, chunked, **httplib_request_kw)
    386         else:
--> 387             conn.request(method, url, **httplib_request_kw)
    388

/usr/local/python3.7/lib/python3.7/http/client.py in request(self, method, url, body, headers, encode_chunked)
   1228         """Send a complete request to the server."""
-> 1229         self._send_request(method, url, body, headers, encode_chunked)
   1230

/usr/local/python3.7/lib/python3.7/http/client.py in _send_request(self, method, url, body, headers, encode_chunked)
   1274             body = _encode(body, 'body')
-> 1275         self.endheaders(body, encode_chunked=encode_chunked)
   1276

/usr/local/python3.7/lib/python3.7/http/client.py in endheaders(self, message_body, encode_chunked)
   1223             raise CannotSendHeader()
-> 1224         self._send_output(message_body, encode_chunked=encode_chunked)
   1225

/usr/local/python3.7/lib/python3.7/http/client.py in _send_output(self, message_body, encode_chunked)
   1015         del self._buffer[:]
-> 1016         self.send(msg)
   1017

/usr/local/python3.7/lib/python3.7/http/client.py in send(self, data)
    955             if self.auto_open:
--> 956                 self.connect()
    957             else:

~/work/python3env/lib/python3.7/site-packages/urllib3/connection.py in connect(self)
    183     def connect(self):
--> 184         conn = self._new_conn()
    185         self._prepare_conn(conn)

~/work/python3env/lib/python3.7/site-packages/urllib3/connection.py in _new_conn(self)
    168             raise NewConnectionError(
--> 169                 self, "Failed to establish a new connection: %s" % e
    170             )

NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7ffff46dd160>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

MaxRetryError                             Traceback (most recent call last)
~/work/python3env/lib/python3.7/site-packages/requests/adapters.py in send(self, request, stream, timeout, verify, cert, proxies)
    448                     retries=self.max_retries,
--> 449                     timeout=timeout
    450                 )

~/work/python3env/lib/python3.7/site-packages/urllib3/connectionpool.py in urlopen(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)
    719             retries = retries.increment(
--> 720                 method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
    721             )

~/work/python3env/lib/python3.7/site-packages/urllib3/util/retry.py in increment(self, method, url, response, error, _pool, _stacktrace)
    435         if new_retry.is_exhausted():
--> 436             raise MaxRetryError(_pool, url, error or ResponseError(cause))
    437

MaxRetryError: HTTPConnectionPool(host='localhost', port=54235): Max retries exceeded with url: /razer/chromasdk (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffff46dd160>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

ConnectionError                           Traceback (most recent call last)
<ipython-input-8-ec47e7bd5d0b> in <module>
----> 1 app = ChromaPython.ChromaApp(Info)

~/work/python3env/lib/python3.7/site-packages/ChromaPython/ChromaApp.py in __init__(self, Info)
     20                 "category": Info.Category
     21             }
---> 22             response = requests.post(url=url, json=data)
     23             self.SessionID, self.URI = response.json()['sessionid'], response.json()['uri']
     24             self.heartbeat = Heartbeat(self.URI)

~/work/python3env/lib/python3.7/site-packages/requests/api.py in post(url, data, json, **kwargs)
    114     """
    115
--> 116     return request('post', url, data=data, json=json, **kwargs)
    117
    118

~/work/python3env/lib/python3.7/site-packages/requests/api.py in request(method, url, **kwargs)
     58     # cases, and look like a memory leak in others.
     59     with sessions.Session() as session:
---> 60         return session.request(method=method, url=url, **kwargs)
     61
     62

~/work/python3env/lib/python3.7/site-packages/requests/sessions.py in request(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)
    531         }
    532         send_kwargs.update(settings)
--> 533         resp = self.send(prep, **send_kwargs)
    534
    535         return resp

~/work/python3env/lib/python3.7/site-packages/requests/sessions.py in send(self, request, **kwargs)
    644
    645         # Send the request
--> 646         r = adapter.send(request, **kwargs)
    647
    648         # Total elapsed time of the request (approximately)

~/work/python3env/lib/python3.7/site-packages/requests/adapters.py in send(self, request, stream, timeout, verify, cert, proxies)
    514                 raise SSLError(e, request=request)
    515
--> 516             raise ConnectionError(e, request=request)
    517
    518         except ClosedPoolError as e:

ConnectionError: HTTPConnectionPool(host='localhost', port=54235): Max retries exceeded with url: /razer/chromasdk (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffff46dd160>: Failed to establish a new connection: [Errno 111] Connection refused')
```

Long story short, it seems to be lookding for /razer/chromasdk but since I didn't manually installed any driver it doesn't exists. 
Therefore my question are:

1. whether I can install something manually and will it solve the issue?
2. will this be supported in linux? 
3. can I set a custom path for the SDK?

Thanks,
Ron
